src/applications/settings/webui/vue-app/src/views/HomeView.vue

```diff
@@ -1,11 +1,24 @@
 <template>
   <div class="row">
+    <h4 class="title">General</h4>
     <div class="twelve columns">
-      <label>Device Name</label>
-      <input class="u-full-width" v-model="settings.deviceName" />
+      <label for="deviceName">Device Name</label>
+      <input id="deviceName" class="u-full-width" type="text" v-model="settings.deviceName" minlength="1" />
     </div>
-    <button class="button-primary" @click="save">Save</button>
-    <div>{{ message }}</div>
+  </div>
+
+  <div class="row">
+    <h4 class="title">Wi-Fi Settings</h4>
+    <div class="twelve columns">
+      <label for="wifiSSID">SSID (Wi-Fi Name)</label>
+      <input id="wifiSSID" class="u-full-width" type="text" v-model="settings.wifiSSID" minlength="1" />
+    </div>
+    <div class="twelve columns">
+      <label for="wifiPassword">Password</label>
+      <input id="wifiPassword" class="u-full-width" type="password" v-model="settings.wifiPassword" minlength="8" />
+    </div>
+    <button class="button-primary" @click="validateAndSave">Save</button>
+    <div v-html="message"></div>
```

Comment:
Please rename `wifiSSID` to `webuiWifiSSID` and `wifiPassword` to `webuiWifiPassword`.